Controllers/HomeController.cs

```diff
@@ -3,24 +3,47 @@
 using System.Diagnostics;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using Recitopia.Data;
 using Recitopia.Models;
+using Microsoft.EntityFrameworkCore;
 
 namespace Recitopia.Controllers
 {
     public class HomeController : Controller
     {
         private readonly ILogger<HomeController> _logger;
+        private readonly RecitopiaDBContext _recitopiaDbContext;
 
-        public HomeController(ILogger<HomeController> logger)
+        public HomeController(ILogger<HomeController> logger, RecitopiaDBContext recitopiaDbContext)
         {
             _logger = logger;
```

Comment:
Check for null and throw `ArgumentNullException` if so

Reply:
Is there more to be done here?